chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -225,10 +225,10 @@ fi
 
 # for making HA on shared NFS backend storage work
 add_group glance 200
-add_user glance 200 glance
 add_group qemu 201
-add_user qemu 201 kvm
 add_group kvm 202
+add_user glance 200 glance
+add_user qemu 201 kvm
 
```

Comment:
wouldn't it make sense to keep group/user add order so that the relationship is easily visible?